department-of-veterans-affairs/caseflow
# Hearing Prep - fetch appeals that were remanded

Sshow appeals on the upcoming hearing days page with no decision date and with dispositions: "3" Remanded and "L" Manlincon remands. Exclude all of the other dispositions

Steps to validate:
1. Find appeals that have a decision date and dispositions: "3" Remanded or "L" Manlincon remands
2. Ensure hearings show up in the upcoming dockets page and ensure appeal streams and issues show up in the worksheet page
